adxl345/example/examples.go

```diff
@@ -0,0 +1,96 @@
+// Copyright 2023 The Periph Authors. All rights reserved.
+// Use of this source code is governed under the Apache License, Version 2.0
+// that can be found in the LICENSE file.
+
+package example
```

Comment:
You can use `package adxl345_test` and it will show up great in the documentation.

Reply:
Done